### ISSUE TITLE
Adapt southeast asia conference gallery

### ISSUE THREAD

**Issue by joewiz** [HistoryAtState/hsg-shell#97]
history.state.gov had a gallery of photos for a conference - see https://history.state.gov/gallery/southeast-asia-conference. 

We have a link to this at http://localhost:8080/exist/apps/hsg-shell/conferences/2010-southeast-asia/photos. Please adapt this gallery code from `/db/history/modules/gallery.xqm`, attached here for convenience: [gallery.xqm.zip](https://github.com/eXistSolutions/hsg-shell/files/83074/gallery.xqm.zip). When complete, the link from http://localhost:8080/exist/apps/hsg-shell/conferences/2010-southeast-asia/photos should work, and the gallery should operate like the current gallery at https://history.state.gov/gallery/southeast-asia-conference.

**Comment by joewiz** [comment]
While it would be nice to have the functionality of the original browser (as attached in the description above), a very basic display of the images would suffice. It would be one of the last remaining items before we can remove the "beta" label from the site.

**Comment by joewiz** [comment]
The URLs that should be visible after completing this are:

- /gallery
- /gallery/southeast-asia-conference
- /gallery/southeast-asia-conference?group=iconic
- /gallery/southeast-asia-conference?group=media
- /gallery/southeast-asia-conference?group=notable-people
- /gallery/southeast-asia-conference?group=on-the-ground
- /gallery/southeast-asia-conference?group=peace-negotiations
- /gallery/southeast-asia-conference?group=protest

**Comment by joewiz** [comment]
Here's a link to the Internet Archive snapshot of this resource for reference: 

https://web.archive.org/web/20130725101226/https://history.state.gov/gallery/southeast-asia-conference

**Comment by joewiz** [comment]
Closed by https://github.com/HistoryAtState/conferences/commit/76cfc3fbe9909187d06a520d28b6ea72fbf3b765.
